### DISCUSSION TITLE
Windows Home Server 2011 Updates

### DISCUSSION THREAD

**Discussion by slendertech** [LegacyUpdate/LegacyUpdate#75] [Ideas]
So, just for fun, I tried installing Legacy Update on my Windows Home Server 2011 machine, as it is a fresh install and could use some updates. As expected, it didn't work. Could you please make Legacy Update compatible with Windows Home Server 2011? Believe it or not, even though Windows Home Server 2011 is based on Windows Server 2008 R2, which is based on Windows 7, which Legacy update both support, it still does not work. It just gives you the error that the update server request has been terminated and that it is commonly caused by the update certificate being out of date. **Please add support for Windows Home Server 2011.**

**Comment by kirb** [comment]
Thanks for posting! This would be the same issue as #51.
